storage/src/persistent/mod.rs

```diff
@@ -71,6 +79,28 @@ pub trait MultiInstanceable {
     }
 }
 
+/// Open commit log at a given path.
+pub fn open_main_db<C: RocksDbColumnFactory>(
+    rocks_db: Option<Arc<DB>>,
+    config: &RocksDbConfig<C>,
+    backend_config: TezedgeDatabaseBackendConfiguration,
+) -> Result<TezedgeDatabase, DatabaseError> {
+    //Todo Change this
```

Comment:
@mambisi 
what does it mean?
remove comment or add please TODO-jira what needs to be done